x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/inference.tsx

```diff
@@ -27,43 +27,29 @@ import { FieldsConfig, to, from, EDITOR_PX_HEIGHT } from './shared';
 const { emptyField, isJsonField } = fieldValidators;
 
 const INFERENCE_CONFIG_DOCS = {
-  regression: {
+  documentation: {
     linkLabel: i18n.translate(
-      'xpack.ingestPipelines.pipelineEditor.inferenceForm.inferenceConfigField.regressionLinkLabel',
-      { defaultMessage: 'regression' }
-    ),
-  },
-  classification: {
-    linkLabel: i18n.translate(
-      'xpack.ingestPipelines.pipelineEditor.inferenceForm.inferenceConfigField.classificationLinkLabel',
-      { defaultMessage: 'classification' }
+      'xpack.ingestPipelines.pipelineEditor.inferenceForm.inferenceConfigField.documentationLinkLabel',
+      { defaultMessage: 'documentation' }
     ),
   },
 };
 
-const getInferenceConfigHelpText = (
-  regressionDocsLink: string,
-  classificationDocsLink: string
-): React.ReactNode => {
+function getInferenceConfigHelpText(documentationDocsLink: string): React.ReactNode {
   return (
     <FormattedMessage
       id="xpack.ingestPipelines.pipelineEditor.inferenceForm.inferenceConfigurationHelpText"
-      defaultMessage="Contains the inference type and its options. There are two types: {regression} and {classification}."
+      defaultMessage="Contains the inference type and its options. Refer to the {documentation} for the available configuration options."
       values={{
-        regression: (
-          <EuiLink external target="_blank" href={regressionDocsLink}>
-            {INFERENCE_CONFIG_DOCS.regression.linkLabel}
-          </EuiLink>
-        ),
-        classification: (
-          <EuiLink external target="_blank" href={classificationDocsLink}>
-            {INFERENCE_CONFIG_DOCS.classification.linkLabel}
+        documentation: (
+          <EuiLink external target="_blank" href={documentationDocsLink}>
```

Comment:
nit: I know this was here from before but setting `target="_blank"` defaults to `external={true}`. See [docs](https://elastic.github.io/eui/#/navigation/link#external-links)